apps/decoder_json/lib/decoder/json.ex

```diff
@@ -14,14 +14,12 @@ defmodule Decoder.Json do
   defimpl Decoder do
     def lines_or_bytes(_t), do: :line
 
-    def decode(t, stream) do
-      stream
-      |> Enum.to_list()
+    def decode(_t, messages) do
+      messages
       |> List.flatten()
```

Comment:
Do we still need to flatten the list here?

Reply:
Not sure; our main reason for looking for feedback was wondering what assumptions we can make about the format of the data coming into these decoders? They definitely won't be streams any longer, but can we assume they are all complete and well-formed packets/payloads or could they be split along arbitrary lines by some means?

Reply:
Looks like not. We can get this changed

Reply:
probably safe to go ahead and remove this `List.flatten`